files/en-us/glossary/property/javascript/index.html

```diff
@@ -5,7 +5,7 @@
   - CodingScripting
   - Glossary
 ---
-<p>A <strong>JavaScript property</strong> is a characteristic of an object, often describing attributes associated with a data structure.</p>
+<p>A <strong>JavaScript property</strong> is a part of an object, that associates a key and a value.</p>
```

Comment:
```suggestion
<p>A <strong>JavaScript property</strong> is a characteristic of an object, often describing attributes associated with a data structure.</p>
```

Whilst I understand why you made this change, I think the original is easier understood. This phrasing is to blunt and will confuse readers 👍

Reply:
Hi, @Rumyra I picked up the `property` definition from here : 

https://262.ecma-international.org/6.0/#sec-property

If that sounds blunt then I will remove it for sure

Reply:
:) yes I see - I think the original here should be kept, as we try to translate specs so they are better understood by developers - thanks!

Reply:
For what it's worth, I'd also point out that JS properties are just as likely to be indicators of object state as they are to hold data.